reference/composedshader.md

```diff
@@ -0,0 +1,105 @@
+## ComposedShader
+
+```
+ComposedShader {
+  vrmlField MFNode fields []
+  vrmlField MFNode parts  []
```

Comment:
We are already not using the exact same X3D standard. ``fields`` doesn't mean anything and for users it easier to understand what this field should contain if it has a different name.